<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Remote_Control</class>
 <widget class="QMainWindow" name="Remote_Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote_Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QStackedWidget" name="wd_app">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>-20</y>
      <width>511</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="wd_login">
     <widget class="QFrame" name="fr_login_head">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>451</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lb_login_header">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>22</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Đăng nhập Client</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="img_login_logo">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>211</width>
         <height>171</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resources.qrc">:/Img/Img/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="fr_login_info">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>421</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="lb_login_port">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Port:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lin_login_client">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lb_login_ip">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>IP:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lin_login_ip">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lb_login_client">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tên Client:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lin_login_port">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_login_connect">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>460</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(36, 255, 97);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>CONNECT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login_change_color">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>☀️</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wd_home">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>91</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <layout class="QVBoxLayout" name="ver_home_menu">
       <item>
        <widget class="QPushButton" name="btn_home_screen">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Chụp màn
hình</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_home_key">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Bắt phím
nhấn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_home_folder">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Quản lý
thư mục</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_home_app">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Quản lý
App</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_home_process">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Quản lý
tiến trình</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="wd_home_work">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>361</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(203, 226, 255);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="wd_home_menu">
       <widget class="QLabel" name="img_home_menu_logo">
        <property name="geometry">
         <rect>
          <x>-2</x>
          <y>25</y>
          <width>361</width>
          <height>211</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/Img/Img/cmsnBac.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="lb_home_menu_hi">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>341</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>App hay quá nè!!! 
Mời bạn thử nha ❤️❤️❤️</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="wd_home_folder">
       <widget class="QTreeView" name="tv_home_folder">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>361</width>
          <height>411</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 226, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
       <widget class="QFrame" name="fr_home_folder_search">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>341</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 226, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>341</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="hor_home_folder_search">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QLineEdit" name="lin_home_folder_path">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_home_folder_submit">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(203, 255, 169);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>SUBMIT</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QListView" name="lv_home_folder_all">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>361</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <zorder>lv_home_folder_all</zorder>
       <zorder>tv_home_folder</zorder>
       <zorder>fr_home_folder_search</zorder>
      </widget>
      <widget class="QWidget" name="wd_home_app">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 255, 255);</string>
       </property>
       <widget class="QTabWidget" name="twd_home_app">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>361</width>
          <height>411</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 226, 255);</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_home_app_all">
         <attribute name="title">
          <string>Tất cả App</string>
         </attribute>
         <widget class="QListView" name="lv_home_app_all">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>361</width>
            <height>381</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_home_app_run">
         <attribute name="title">
          <string>Chạy App</string>
         </attribute>
         <widget class="QListView" name="lv_home_app_run">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>361</width>
            <height>381</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_home_app_close">
         <attribute name="title">
          <string>Đóng App</string>
         </attribute>
         <widget class="QListView" name="lv_home_app_close">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>361</width>
            <height>381</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="wd_home_process">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <widget class="QTabWidget" name="twd_home_process">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>361</width>
          <height>411</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 226, 255);</string>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_home_process_all">
         <attribute name="title">
          <string>Tất cả Process</string>
         </attribute>
         <widget class="QListView" name="lv_home_process_all">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>361</width>
            <height>381</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_home_process_run">
         <attribute name="title">
          <string>Chạy Process</string>
         </attribute>
         <widget class="QListView" name="lv_home_process_run">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>361</width>
            <height>381</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_home_process_close">
         <attribute name="title">
          <string>Đóng Process</string>
         </attribute>
         <widget class="QListView" name="lv_home_process_close">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>361</width>
            <height>381</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="wd_home_screen">
       <widget class="QPushButton" name="btn_home_screen_link">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>350</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Tới Đường Dẫn</string>
        </property>
       </widget>
       <widget class="QLabel" name="img_home_screen">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>361</width>
          <height>221</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/Img/Img/M10.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="wd_home_key">
       <widget class="QPlainTextEdit" name="plt_home_key">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>341</width>
          <height>251</height>
         </rect>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
       <widget class="QLabel" name="lb_home_key_press">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>290</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>Key Press</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="lb_home_head">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>22</pointsize>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string>MENU</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_home_back">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>🔙</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_File">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icon/Icon/add.png</normaloff>:/Icon/Icon/add.png</iconset>
   </property>
   <property name="text">
    <string>Add File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete_File">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icon/Icon/remove.png</normaloff>:/Icon/Icon/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete File</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icon/Icon/rename.png</normaloff>:/Icon/Icon/rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionCopy_File">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icon/Icon/copy.png</normaloff>:/Icon/Icon/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste_File">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icon/Icon/paste.png</normaloff>:/Icon/Icon/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lin_login_client</tabstop>
  <tabstop>lin_login_ip</tabstop>
  <tabstop>lin_login_port</tabstop>
  <tabstop>btn_login_connect</tabstop>
  <tabstop>btn_home_screen</tabstop>
  <tabstop>btn_home_key</tabstop>
  <tabstop>btn_home_folder</tabstop>
  <tabstop>btn_home_app</tabstop>
  <tabstop>btn_home_process</tabstop>
  <tabstop>tv_home_folder</tabstop>
  <tabstop>lin_home_folder_path</tabstop>
  <tabstop>btn_home_folder_submit</tabstop>
  <tabstop>plt_home_key</tabstop>
  <tabstop>btn_home_screen_link</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
